tools/cldr/cldr-to-icu/pom.xml

```diff
@@ -83,7 +83,7 @@
         <dependency>
             <groupId>com.ibm.icu</groupId>
             <artifactId>icu4j</artifactId>
-            <version>74.1-SNAPSHOT</version>
+            <version>74.1</version>
```

Comment:
This is the version of icu used by the cldr-icu tooling. I do not think any change in 74.2 would affect the behavior of that tooling; and since the vision number had not been updated when I did the integration (so there was no icu4j artifact for 74.2), I could not use 74.2 anyway. No need for https://github.com/unicode-org/icu/pull/2728 to touch this because we will not be doing another cldr-icu integration for 74.2.